### PR TITLE
Add lsstdesc-env recipe

### PR DESCRIPTION
**Pull request by heather999**
This is a metapackage for the LSSTDESC (Dark Energy Science Collaboration) to define a common user environment.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/lsstdesc-env/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
+{% set name = "lsstdesc-env" %}
+{% set date = datetime.datetime.utcnow().strftime('%Y.%m.%d') %}
+{% set build_number = 0 %}
+{% set version = date ~ '_' ~ build_number %}
```
Comment (beckermr) [inline recipes/lsstdesc-env/meta.yaml:4]: Use a `.` here.

File: recipes/lsstdesc-env/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    - seaborn
+    - snappy
+    - sphinx
+    - tabulate
+    - treecorr
+    - yaml
+
+test:
+  imports:
+    - pyccl
```
Comment (beckermr) [inline recipes/lsstdesc-env/meta.yaml:78]: This is going to confuse other bits of code. Let's use a hello world command.

**Review by beckermr — CHANGES_REQUESTED** [review]

File: recipes/lsstdesc-env/meta.yaml (hunk 1 above)
Comment (beckermr) [inline recipes/lsstdesc-env/meta.yaml:4]:
```suggestion
{% set version = date ~ '.' ~ build_number %}
```

File: recipes/lsstdesc-env/meta.yaml (hunk 2 above)
Comment (beckermr) [inline recipes/lsstdesc-env/meta.yaml:78]:
```suggestion
  commands:
    - echo 'it works!'
```

File: recipes/lsstdesc-env/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    - echo 'it works!'
+
+
+about:
+  home: https://github.com/conda-forge/lsstdesc-env-feedstock
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: Metapackage to install the DESC's common software environment.
+
```
Comment (beckermr) [inline recipes/lsstdesc-env/meta.yaml:87]:
```suggestion
```

**Review by beckermr — APPROVED** [review]

**Comment by beckermr** [comment]
@conda-forge/staged-recipes this is a noarch metapackage for use elsewhere - Ready for review!

File: recipes/lsstdesc-env/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    - seaborn
+    - snappy
+    - sphinx
+    - tabulate
+    - treecorr
+    - yaml
+
+test:
+  commands:
+    - echo 'it works!'
```
Comment (ocefpaf) [inline recipes/lsstdesc-env/meta.yaml:78]: Nice test!

**Pull request merged**